Jenkinsfile

```diff
@@ -1,50 +1,202 @@
     projectName = 'vendor-headsets'
-    manifest = directoryManifest('dist')
-    buildType = { (env.BRANCH_NAME == 'master' || env.BRANCH_NAME.startsWith('release/')) ? 'MAINLINE' : 'FEATURE' }
-    publishPackage = { 'dev' }
-    testJob = null
+    team = 'Genesys Client Media (WebRTC)'
+    mailer = 'genesyscloud-client-media@genesys.com'
+    chatGroupId = '763fcc91-e530-4ed7-b318-03f525a077f6'
+
+    nodeVersion = '14.x'
+    buildType = getBuildType
+
+    manifest = customManifest('./dist') {
+      sh('node ./create-manifest.js')
+      readJSON(file: 'dist/manifest.json')
+    }
+
+    deployConfig = [
+      dev : 'always',
+      test : 'always',
+      prod : 'always',
+      'fedramp-use2-core': 'always'
+    ]
+
+    autoSubmitCm = true
+
+    testJob = 'no-tests' // see buildStep to spigot tests
+
+    ciTests = {
+        println("""
+========= BUILD VARIABLES =========
+ENVIRONMENT  : ${env.ENVIRONMENT}
+BUILD_NUMBER : ${env.BUILD_NUMBER}
+BUILD_ID     : ${env.BUILD_ID}
+BRANCH_NAME  : ${env.BRANCH_NAME}
+APP_NAME     : ${env.APP_NAME}
+VERSION      : ${env.VERSION}
+===================================
+      """)
+
+      sh("""
+        npm i -g npm@7
+        npm run install:all
+        npm run lint
+        npm run test:coverage
+      """)
+    }
+
+    buildStep = {cdnUrl ->
+        sh("""
+            echo 'CDN_URL ${cdnUrl}'
+            npm run compile:module
+        """)
+    }
 
     buildStep = {
-        sh('''
-            export CDN_URL="$(npx cdn --ecosystem pc --name \$APP_NAME --build \$BUILD_ID --version \$VERSION)"
-            echo "CDN_URL: \$CDN_URL"
-            npm run install:all && npm run compile:module && npm run build && npm run lint && npm run test:coverage
-        ''')
+        sh("""
+            npm run build
+        """)
     }
 
-    snykConfig = {
-        return [
+    onSuccess = {
+       sh("""
+            echo "=== root folder ==="
+            ls -als ./
+
+            echo "=== Printing manifest.json ==="
+            cat ./manifest.json
+
+            echo "=== Printing package.json ==="
+            cat ./package.json
+
+            echo "=== dist folder ==="
+            ls -als dist/
+
+            echo "=== Printing dist/deploy-info.json ==="
+            cat ./dist/deploy-info.json
+
+            # echo "=== Printing dist/package.json ==="
+            # cat ./dist/package.json
+        """)
+
+        snykConfig = {
+          return [
             organization: 'genesys-client-media-webrtc',
             project: 'softphone-vendor-headsets'
-        ]
-    }
+          ]
+        }
 
-    cmConfig = {
-        return [
+        cmConfig = {
+          return [
             managerEmail: 'purecloud-client-media@genesys.com',
             rollbackPlan: 'Patch version with fix',
-            testResults: 'https://jenkins.ininica.com/job/valve-webrtcsdk-tests-test/',
+            testResults: 'https://jenkins.inindca.com/job/valve-webrtcsdk-tests-test/',
```

Comment:
The _"I"_ stands for "infra" (ie. infrastructure) so `ininica.com` is correct. However, this url is for volt. Since the headset lib doesn't have a dedicated test job, just link to the `master` jenkins job.